src/EventHub/EventHub/Cmdlets/Namespace/SetAzureEventHubNamespace.cs

```diff
@@ -86,8 +97,40 @@ public class SetAzureEventHubNamespace : AzureEventHubsCmdletBase
         /// </summary>
         [Parameter(Mandatory = false, ParameterSetName = AutoInflateParameterSet, HelpMessage = "enabling or disabling Kafka for namespace")]
         [Parameter(Mandatory = false, ParameterSetName = NamespaceParameterSet, HelpMessage = "enabling or disabling Kafka for namespace")]
+        [Parameter(Mandatory = false, ParameterSetName = IdentityUpdateParameterSet, HelpMessage = "enabling or disabling Kafka for namespace")]
         public SwitchParameter EnableKafka { get; set; }
 
+        /// <summary>
+        /// Indicates whether Identity is enabled.
+        /// </summary>
+        [Parameter(Mandatory = false, ParameterSetName = AutoInflateParameterSet, HelpMessage = "enabling or disabling Identity for namespace")]
+        [Parameter(Mandatory = false, ParameterSetName = NamespaceParameterSet, HelpMessage = "enabling or disabling Identity for namespace")]
+        [Parameter(Mandatory = false, ParameterSetName = IdentityUpdateParameterSet, HelpMessage = "enabling or disabling Identity for namespace")]
+        public SwitchParameter Identity { get; set; }
+
+        /// <summary>
+        /// Sets Identity None or other than SystemAssigned .
+        /// </summary>
+        [Parameter(Mandatory = false, ParameterSetName = AutoInflateParameterSet, HelpMessage = "User defined Identity or None")]
+        [Parameter(Mandatory = false, ParameterSetName = NamespaceParameterSet, HelpMessage = "User defined Identity or None")]
+        [PSArgumentCompleter("None")]
+        public string IdentityUserDefined { get; set; }
+
+        /// <summary>
+        /// Key Source
+        /// </summary>
+        [Parameter(Mandatory = false, ParameterSetName = AutoInflateParameterSet, HelpMessage = "Key Source")]
+        [Parameter(Mandatory = false, ParameterSetName = NamespaceParameterSet, HelpMessage = "Key Source ")]
+        [PSArgumentCompleter("Microsoft.KeyVault")]
+        public string KeySource { get; set; }
+
+        /// <summary>
+        /// Key Source
+        /// </summary>
+        [Parameter(Mandatory = false, ParameterSetName = AutoInflateParameterSet, HelpMessage = "List of Key Properties, @(@(KeyName,KeyVaultUri,Keyversion),@(KeyName,KeyVaultUri,Keyversion))")]
+        [Parameter(Mandatory = false, ParameterSetName = NamespaceParameterSet, HelpMessage = "List of Key Properties, @(@(KeyName,KeyVaultUri,Keyversion),@(KeyName,KeyVaultUri,Keyversion))")]
+        public List<string []> KeyProperties { get; set; }
```

Comment:
It's nessary to follow the naming guideline of powershell. You can see some suggestion from [here](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/cmdlet-best-practices.md#specific-noun-and-noun-singularity).

Reply:
changed to KeyProperty